### PR TITLE
feat(@desktop/onboarding): Biometrics screen updates

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Closes #5602

### What does the PR do

- Updates text on the Biometrics screen
- Updates img on the Biometrics screen
- Hides back button on the Biometrics screen

### Affected areas

onboarding, mac only

### Screenshot of functionality

image

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

**Review by alexandraB99 — CHANGES_REQUESTED** [review]

File: ui/app/AppLayouts/Onboarding/views/TouchIDAuthView.qml [hunk 1]
```diff
@@ -76,7 +79,7 @@ OnboardingBasePage {
             StatusButton {
                 id: button
                 Layout.alignment: Qt.AlignHCenter
-                text: qsTr("Yes, use TouchID ")
+                text: qsTr("Yes, use Touch ID ")
```
Comment (saledjenic) [inline ui/app/AppLayouts/Onboarding/views/TouchIDAuthView.qml:82]: @MishkaRogachev we don't need `" "` (space) at the end.

**Review by saledjenic — APPROVED** [review]

File: ui/app/AppLayouts/Onboarding/controls/OnboardingBasePage.qml [hunk 2]
```diff
@@ -15,6 +18,7 @@ Page {
     }
 
     StatusRoundButton {
+        id: backButton
```
[the remark this answers is not in the document]
Comment (MishkaRogachev) [reply ui/app/AppLayouts/Onboarding/controls/OnboardingBasePage.qml:21]: Hmmm. I've just removed back button from TouchIDAuthView.qml, but button still present on GenKeyView. Is it correct?

[the remark this answers is not in the document]
Comment (MishkaRogachev) [reply ui/app/AppLayouts/Onboarding/controls/OnboardingBasePage.qml:21]: Fixed!

**Review by iurimatias — APPROVED** [review]
Screen Shot 2022-05-09 at 4 51 50 PM

biometrics still work too

**Pull request merged**